src/makielayout/types.jl

```diff
@@ -1152,15 +1152,32 @@ const CHECKMARK_BEZIER = scale(BezierPath(
     end
 end
 
+"""
+A switch with two states.
+
+## Constructors
+
+```julia
+Toggle(fig_or_scene; kwargs...)
+```
+
+## Examples
+
+```julia
+t_horizontal = Toggle(fig[1, 1])
+t_vertical = Toggle(fig[2, 1], orientation = pi/2)
+```
```

Comment:
Sweet!  Would also be nice to add a small `on` to demonstrate how it's done, if possible.